package.json

```diff
@@ -44,32 +47,92 @@
             "./"
           ],
           "alias": {
-            "@reactioncommerce/reaction-collections": "./imports/plugins/core/collections",
             "@reactioncommerce/reaction-components": "./imports/plugins/core/components/lib",
-            "@reactioncommerce/reaction-router": "./imports/plugins/core/router/lib",
-            "@reactioncommerce/reaction-ui": "./imports/plugins/core/ui/client/components",
             "underscore": "lodash"
           }
         }
       ]
     ],
-    "presets": [
-      "env",
-      "react",
-      "stage-2"
-    ]
+    "env": {
```

Comment:
Will it work if we just don't have these babel configs in this package.json. Won't they be just picked up from the reaction's package.json?